Build error when using `--features="serde"`

When trying to build v0.16.0-rc2 with the `serde` feature, compilation fails:

```
vscode ➜ /workspaces/bevy_ecs_tilemap (main) $ cargo build --features="serde"
   Compiling bevy_ecs_tilemap v0.16.0-rc.2 (/workspaces/bevy_ecs_tilemap)
error[E0277]: the trait bound `bevy::prelude::Entity: map::_::_serde::Serialize` is not satisfied
   --> src/map.rs:56:38
    |
56  | #[cfg_attr(feature = "serde", derive(serde::Serialize, serde::Deserialize))]
    |                                      ^^^^^^^^^^^^^^^^ the trait `map::_::_serde::Serialize` is not implemented for `bevy::prelude::Entity`
57  | pub struct TilemapId(pub Entity);
    |                      --- required by a bound introduced by this call
    |
    = note: for local types consider adding `#[derive(serde::Serialize)]` to your `bevy::prelude::Entity` type
    = note: for types from other crates check whether the crate offers a `serde` feature flag
    = help: the following other types implement trait `map::_::_serde::Serialize`:
              &'a T
              &'a mut T
              ()
              (T,)
              (T0, T1)
              (T0, T1, T2)
              (T0, T1, T2, T3)
              (T0, T1, T2, T3, T4)
            and 426 others
note: required by a bound in `serialize_newtype_struct`
   --> /usr/local/cargo/registry/src/index.crates.io-1949cf8c6b5b557f/serde-1.0.215/src/ser/mod.rs:907:21
    |
901 |     fn serialize_newtype_struct<T>(
    |        ------------------------ required by a bound in this associated function
...
907 |         T: ?Sized + Serialize;
    |                     ^^^^^^^^^ required by this bound in `Serializer::serialize_newtype_struct`
    = note: this error originates in the derive macro `serde::Serialize` (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0277]: the trait bound `bevy::prelude::Entity: Deserialize<'_>` is not satisfied
  --> src/map.rs:57:26
   |
57 | pub struct TilemapId(pub Entity);
   |                          ^^^^^^ the trait `Deserialize<'_>` is not implemented for `bevy::prelude::Entity`
   |
   = note: for local types consider adding `#[derive(serde::Deserialize)]` to your `bevy::prelude::Entity` type
   = note: for types from other crates check whether the crate offers a `serde` feature flag
   = help: the following other types implement trait `Deserialize<'de>`:
             &'a [u8]
             &'a std::path::Path
             &'a str
             ()
             (T,)
             (T0, T1)
             (T0, T1, T2)
             (T0, T1, T2, T3)
           and 427 others

error[E0277]: the trait bound `bevy::prelude::Entity: Deserialize<'_>` is not satisfied
    --> src/map.rs:57:26
     |
57   | pub struct TilemapId(pub Entity);
     |                          ^^^^^^ the trait `Deserialize<'_>` is not implemented for `bevy::prelude::Entity`
     |
     = note: for local types consider adding `#[derive(serde::Deserialize)]` to your `bevy::prelude::Entity` type
     = note: for types from other crates check whether the crate offers a `serde` feature flag
     = help: the following other types implement trait `Deserialize<'de>`:
               &'a [u8]
               &'a std::path::Path
               &'a str
               ()
               (T,)
               (T0, T1)
               (T0, T1, T2)
               (T0, T1, T2, T3)
             and 427 others
note: required by a bound in `next_element`
    --> /usr/local/cargo/registry/src/index.crates.io-1949cf8c6b5b557f/serde-1.0.215/src/de/mod.rs:1732:12
     |
1730 |     fn next_element<T>(&mut self) -> Result<Option<T>, Self::Error>
     |        ------------ required by a bound in this associated function
1731 |     where
1732 |         T: Deserialize<'de>,
     |            ^^^^^^^^^^^^^^^^ required by this bound in `SeqAccess::next_element`

For more information about this error, try `rustc --explain E0277`.
error: could not compile `bevy_ecs_tilemap` (lib) due to 3 previous errors
vscode ➜ /workspaces/bevy_ecs_tilemap (main) $ 
```

It does compile when using v0.15 (tested with commit `8ccbd6bff829643a9c6f20ad25367234179886b9`).

Naive fix would be to enable the `serialize` feature on Bevy but do we actually *want* to serialize an `Entity` ? Does it make sense ?

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -13,7 +13,7 @@ exclude = ["assets/*", "screenshots/*"]
 default = ["render"]
 atlas = []
 render = []
-serde = ["dep:serde"]
+serde = ["dep:serde", "bevy/serialize"]
 
 [dependencies]
 bevy = { version = "0.16.0", default-features = false, features = [

